week-07/day-4/UsefulUtilities/UsefulUtilities/Views/Home/index.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>Index</title>
+    </head>
+    <body style="background-color: aqua;">
+        <h1>Useful</h1>
+        <a href="https://localhost:44352/email?email=is@this.valid">valid email</a><br />
```

Comment:
`https://localhost:44352/` shouldn't be included in the URL, just use `/`. This wouldn't work on my machine.